core/helpers/validators/index.ts

```diff
@@ -1,5 +1,5 @@
+export const unicodeAlpha = (value: string): boolean => /^[a-zA-Z_ ]*$/.test(value)
```

Comment:
It is not a proper solution. In the example for the issue, Pablo gave validation for name as an example. Your regex will not pass name with special chars, e.g. it won't pass `Łukasz` because of `Ł`. Old one solved this issue but also allowed for chars like $. 

The target is to support names with special chars (like `Łukasz`) but prevent chars like `$`

Reply:
Fixed